app/components/api/guidance/release_notes/tags_component.rb

```diff
@@ -0,0 +1,48 @@
+module API
+  module Guidance
+    module ReleaseNotes
+      class TagsComponent < ViewComponent::Base
+        class UnknownTagError < StandardError; end
+
+        TAG_MAPPINGS = {
+          "#breaking-change" => "red",
+          "#new-feature" => "green",
+          "#bug-fix" => "yellow",
+          "#new-field" => "turquoise",
+          "#data-update" => "blue",
+          "#contract-closure" => "orange",
+          "#new-course" => "pink",
+          "#production-release" => "purple",
+          "#sandbox-release" => "grey",
+        }.freeze
+
+        attr_accessor :tags
+
+        def initialize(tags:)
+          @tags = tags
+        end
+
+        def all_tags
+          return if tags.blank?
+
+          render_tags
+        end
```

Comment:
I think this can be replaced with [render?](https://viewcomponent.org/guide/conditional_rendering.html), something like:

```
def render?
  tags.any?
end
```